### DISCUSSION TITLE
Localized Payload. Getting all locales in one call.

### DISCUSSION THREAD

**Discussion by ckruppe** [payloadcms/payload#9853] [Q&A] [answered]
Hi, does anyone know i can recieve the collection with no particular locale and therefore texts as an object consisting of all locales defined? Is there an possibillity for that? I need this as requirement for an Desktop/Mobile app that fetches and caches data from Payload and gets only sproradically synced with payload. So it would be nice to have one single json that consists of all defined languages.

**Comment by r1tsuu** [comment, marked as answer]
Hey!
From - https://payloadcms.com/docs/configuration/localization#local-api
> Tip: The REST and Local APIs can return all Localization data in one request by passing 'all' or '*' as the locale parameter. The response will be structured so that field values come back as the full objects keyed for each locale instead of the single, translated value.

Is that what you're looking for?

**Reply by ckruppe** [reply]
Hey thank you very much. Must have overlooked that as i scouted through the docs. :)
Did not expect it under the localApi docs. :)
Its exactly what i needed.
